en/docs/get-started/key-concepts.md

```diff
@@ -23,3 +23,8 @@ Types are used to define the structure of data in Ballerina Integrator. They are
 ## Configurations
 Configurations are used to define the settings and parameters of the integration. They are used to configure the behavior of the integration and its components. 
 Configurations can be used to set up connections, define global variables, and configure other aspects of the integration in different environments.
+
+## Source View
+The source view is a text-based representation of the integration that can be used to view and edit the code directly. It provides a way to see the underlying code that is generated by the graphical interface.
+Switching to the source view allows you to view and edit the code directly, providing more control and flexibility over the integration. 
+Clicking on the `</>` button in the top right corner of the design view will switch to the source view.
```

Comment:
Isn't it better to add a screenshot/gif in here (as this icon is hardly notiable)?  

btw Um not sure whether we've agreed not to add any visual materials in this doc? (as I couldnt find any other images or gifs in this doc)

Reply:
It was not possible to add a screen shot due to kola-> bi renaming was not done. I will update the PR with changes after rename.

Reply:
Added a gif